### ISSUE TITLE
Upgrade from 5.x to 6.x guide

### ISSUE THREAD

**Issue by ivan-nginx** [iissnan/hexo-theme-next#2067]
For anyone, who want to upgrade version 5 to 6 — read instructions here on [English](https://github.com/theme-next/hexo-theme-next/blob/master/docs/UPDATE-FROM-5.1.X.md), [Chinese](https://github.com/theme-next/hexo-theme-next/blob/master/docs/cn/UPDATE-FROM-5.1.X.md) or [Russian](https://github.com/theme-next/hexo-theme-next/blob/master/docs/ru/UPDATE-FROM-5.1.X.md) languages.

**Comment by Neveryu** [comment]
chenese page not found

**Comment by darrenliuwei** [comment]
@Neveryu  中文地址：   https://github.com/theme-next/hexo-theme-next/blob/master/docs/zh-CN/UPDATE-FROM-5.1.X.md

**Comment by jdhao** [comment]
I am curious to know why the new repository is built. Is this repository going to be deprecated?

**Comment by ivan-nginx** [comment]
Yes, u are curious. And u find this issue, but not find [this](https://github.com/iissnan/hexo-theme-next/issues/2061) issue.
And yes, a little bit deprecated. But u can use this repo and it will work. But current maintenance supported only for new repository. It's mean all bugfixes and new features will be in new repo.

**Comment by zbl19962** [comment]
Yes, u are curious. And u find this issue, but not find this issue.

And yes, a little bit deprecated. And u can use this repo and it will work. But current maintenance supported only for new repository. It's mean all bugfixes and new features will be in new repo.
